core/flow/rule_manager.go

```diff
@@ -83,8 +139,19 @@ func onRuleUpdate(rules []*Rule) (err error) {
 		}
 	}()
 
-	m := buildFlowMap(rules)
-
+	resRulesMap := make(map[string][]*Rule)
+	for _, rule := range rules {
+		if err := IsValidRule(rule); err != nil {
+			logging.Error(err, "ignoring invalid flow rule", "rule", rule)
```

Comment:
Actually the only thing we needed is error message (not error itself and its call stack), so maybe we could just extract the error message here (using warn level seems okay).

Reply:
I will update here